scraper/target1337x.go

```diff
@@ -4,62 +4,96 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/PuerkitoBio/goquery"
 	"github.com/qascade/yast/movie"
 
 	colly "github.com/gocolly/colly"
 )
 
+// scrape1337x returns the scraped results in Result type
 func (s *Scraper) scrape1337x(context *QueryContext) (results []Result, err error) {
 
+	var links []string
+	var movielist []movie.Movie
 	results = make([]Result, 0)
```

Comment:
make `var` declaration convention of slices uniform. Change `results = make([]Result,0)` -> `var results []Result`

Reply:
yes ok